### DISCUSSION TITLE
How does one follow talktative people & bots?

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#18606] [Q&A]
Hi,

I recently setup a [server to purely mirror RSS feeds](https://botok.loveisgrief.tk/). It uses a [simple python script](https://gitlab.com/NamingThingsIsHard/net/tootrss) to pull items from RSS feeds and post them to individual accounts. Some of the RSS feeds have a lot of updates and if I follow them, my timeline gets very full - so full that I can't see people.

Unfortunately, lists only work with accounts that I follow

![afbeelding](https://user-images.githubusercontent.com/2829538/172021998-a9ed7034-e57c-491e-9fb8-186adf7fa72a.png)

Is there some way I can still follow my bots (and sometimes other talkative people) without them drowning out low-frequency posters?
Maybe this might be a feature request (allow lists to have accounts one doesn't follow)

**Comment by hello-smile6** [comment]
@LoveIsGrief Could you add NASA's APOD rss feed?

**Reply by nemobis** [reply]
Are you aware of https://botsin.space/@APoD ?
